webpack/environment.js

```diff
@@ -1,6 +1,34 @@
 module.exports = {
     I18N_HASH: 'generated_hash',
     SERVER_API_URL: '',
-    __VERSION__: process.env.hasOwnProperty('APP_VERSION') ? process.env.APP_VERSION : 'DEV',
+    __VERSION__: process.env.hasOwnProperty('APP_VERSION') ? process.env.APP_VERSION : inferVersion(),
     __DEBUG_INFO_ENABLED__: false,
 };
+
+/*
+ * Needed for client compilations with docker-compose, where the 'APP_VERSION' property isn't injected by gradle.
+ *
+ * Returns the inferred APP_VERSION from 'build.gradle', or 'DEV' if this couldn't be retrieved
+ */
+function inferVersion() {
+    let version = 'DEV';
```

Comment:
```suggestion
    let version;
```

Reply:
Would leave this one, as is..
If an error occurs within the `try-catch` section, a `null` value would be returned..

Reply:
Yes,you are right. Did not see that 👍